dev/local-environment-dynamic/modules/cardano.txt

```diff
@@ -19,7 +19,7 @@
       - ./configurations/cardano/keys/op.cert:/keys/node.cert
       - ./configurations/cardano/reward_token_policy.script:/shared/reward_token_policy.script
       - ./configurations/cardano/v-function.script:/shared/v-function.script
-      - ./configurations/cardano/config-pool1.json:/shared/node-1-config.json.base
```

Comment:
Updating genesis hashes in this file is gone in this PR, so we have only final version without .base suffix